radis/lbl/factory.py

```diff
@@ -477,6 +478,12 @@ def __init__(
             cutoff = 0
         self.params.cutoff = cutoff
 
+        # Time Based variables
+        self.verbose = verbose
```

Comment:
@erwanp I'm declaring the CalcTime object here because in `loader.py` the `self.verbose` values remains True and is not updated by the SpectrumFactory value, so it will never be never satisfy `self.verbose>=verbose` condition to print the statement. Declaring here will store the updated value in self.verbose thus we don't have to send it as an argument in `start` or `stop`

Reply:
Are you sure ? SpectrumFactory inherits from the Loader, so Loader.verbose should actually be the same as SpectrumFactory.verbose ?

Reply:
If you see the line [here](https://github.com/radis/radis/blob/4ebfa27ac85dde933df1f523b401ba8130c5b389/radis/lbl/loader.py#L555) we are specify `self.verbose = True`. But if I remove this line so it should automatically take the value from `SF.__init__` , in that case on declaring `self.time = CalcTime(self.verbose)` in `DatabankLoader.__init__`, it gives the following error:

  ```
File "/home/pipebomb/Desktop/radis/radis/lbl/loader.py", line 620, in __init__
    self.time = CalcTime(self.verbose)

AttributeError: 'SpectrumFactory' object has no attribute 'verbose'
```
which is indicating self.verbose hasn't been declared in `self` object yet.

Reply:
I see! That's because CalcTimer keeps the verbose level it had when initialized. 
You could reset this at the beginning of a calculatino ; for instance [here for eq_spectrum](https://github.com/radis/radis/blob/develop/radis/lbl/factory.py#L653). It would also allow you to change the verbose level in between 2 calculations of a same Factory and immediatly see the changes. 

Another option would be to fully Reset the Timer, creating a new instance of the CalcTimer class, at the beginning of the calculation. 
- you can init it with the current verbose level of the Factory.
- no need to copy() the output when storing them in the Spectrum conditions, as I mentionned in another comment. 
- makes sure all keys are properly reset in between 2 calculations